package.json

```diff
@@ -2,7 +2,7 @@
   "name": "markdown-it-link-attributes",
   "version": "2.1.0",
   "description": "A markdown-it plugin to configure the attributes for links",
-  "main": "index.js",
+  "main": "dist/markdown-it-link-attributes.min.js",
```

Comment:
IMO, this is not the right solution. Many build tools prefer unminified code so it can all be minified together. 

This could constitute a breaking change. 

I’d rather just document in the readme that a browser ready version exists at the dist path and can be required via npm.

Reply:
Ok, removed this change and added note about importing browser ready version to readme